code/modules/mob/living/carbon/human/life/handle_environment.dm

```diff
@@ -96,3 +83,33 @@
 		return HEAT_DAMAGE_LEVEL_3
 	else
 		return 0
+
+/mob/living/carbon/human/proc/get_thermal_loss(var/datum/gas_mixture/environment)
+	var/loc_temp = get_loc_temp(environment)
+	if(loc_temp < bodytemperature)
+		// We're going to try and just use exposed area(temperature difference)/cold divisor, and assume we're only conducting.
+		var/thermal_loss = (1-get_cold_protection())  				// How much of your skin is exposed.
+		if(environment.total_moles > 103.934 || !IS_SPACE_COLD)
```

Comment:
(turns out it's already a define but I searched up the number and it didn't turn up because it's an equation or whatever)